### DISCUSSION TITLE
I tried to config the wiki.js from nginx and I got some errors

### DISCUSSION THREAD

**Discussion by DanitRosenberg** [requarks/wiki#3479] [Help / Questions]
Here you can see the errors I got 

This is the configuration - 

    location /wiki {
        proxy_set_header    X-Forwarded-Host $host;
        proxy_set_header    X-Forwarded-Server $host;
        proxy_set_header    X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header    X-Real-IP $remote_addr;
        proxy_pass          http://drptl-knowledge:8080/;
        proxy_http_version  1.1;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection "upgrade";
        proxy_set_header    X-Forwarded-Proto "https";
        proxy_next_upstream error timeout http_502 http_503 http_504;
        proxy_redirect      off;

    }

Thanks for ur help!

**Comment by sattitan19** [comment]
assuming the ngnix is on same server as the wikijs is installed...this should be local host..
    proxy_pass          http://drptl-knowledge:8080/; ->     proxy_pass          http://localhost:8080;

also no trailing /

**Comment by DanitRosenberg** [comment]
The nginx server is another one... the drptl-knowledge server is the wiki server - remote server

**Comment by sattitan19** [comment]
as per the documentation / in the end should not be there..https://docs.nginx.com/nginx/admin-guide/security-controls/securing-http-traffic-upstream/
try removing it

**Comment by DanitRosenberg** [comment]
I tried to remove . Unfortunately, I got the same errors...
